docs/index.md

```diff
@@ -6,9 +6,9 @@ title: Haal Centraal BRP Personen Bevragen
 
 ![lint oas]({{ site.repoUrl }}/workflows/lint-oas/badge.svg)
 
-API voor het zoeken en raadplegen van actuele personen, partners, ouders en kinderen uit de basisregistratie personen (BRP), inclusief de registratie niet-ingezeten (RNI). De start van de pilot {{ site.apiname }} is gepland in maart 2023.
+API voor het zoeken en raadplegen van actuele personen, partners, ouders en kinderen uit de basisregistratie personen (BRP), inclusief de registratie niet-ingezeten (RNI). De pilot {{ site.apiname }} is gestart.
```

Comment:
Ik zou "De pilot is gestart" weglaten. Afgesproken is om in de communicatie wordt het woord "pilot"  zoveel mogelijk weg te laten omdat dit verwarring veroorzaakt (Wel/geen productie?)